content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1363 @@
+
+Once the setup is ready, let's open the Out-of-the-box page. Use the command line window and launch the TCP port forwarding setup command:
+
+```bash
+adb forward tcp:8080 tcp:80
+```
+
+Now open your browser, go to [http://localhost:8080](http://localhost:8080), and the Out-of-the-box dashboard will appear.
+
+![Out-of-the-box Home Screen](assets/OOTB_homepage_init.png "Out-of-the-box Home Screen")
+
+On the Out-of-the-box page, the **SYSTEM INFO** button is located at the bottom left.
+
+![Out-of-the-box System Info](assets/OOTB_system_info.png "Out-of-the-box System Info")
+
+It will show you different types of information about the Portenta X8, including hardware/software information, network status, and active containers. 
+
+#### Wi-Fi® Configuration
+
+Click **Wi-Fi® Connection** to start configuring your network connectivity. Otherwise, you can connect your Portenta X8 to the Internet through an Ethernet cable using a USB-C® hub with an RJ45 port or a Portenta Carrier. In this tutorial, Wi-Fi® connectivity will be used.
+
+![Out-of-the-box Wi-Fi® Settings](assets/OOTB_homepage_Wifi.png "Out-of-the-box Wi-Fi® Settings")
+
+Choose **WiFi Network** option to continue network connectivity configuration with Wi-Fi® connectivity.
+
+![Out-of-the-box Wi-Fi® Network Selection](assets/OOTB_wifi_option.png "Out-of-the-box Wi-Fi® Network Selection")
+
+Select your Wi-Fi® SSID. You can select a network from the available list or introduce your SSID manually.
+
+![Out-of-the-box Wi-Fi® SSID set-up](assets/OOTB_wifi_selection.png "Out-of-the-box Wi-Fi® SSID set-up")
+
+Type your Wi-Fi® password.
+
+![Out-of-the-box Wi-Fi® password set-up](assets/OOTB_wifi_SSID.png "Out-of-the-box Wi-Fi® password set-up")
+
+Once connected, you will get a notification confirming your Portenta X8 has connected to the selected network, and its LED will start blinking green.
+
+Moreover, you can check the network you are connected to in the bottom left section of this dashboard.
+
+![Out-of-the-box Wi-Fi® connection successful](assets/OOTB_wifi_connected.png "Out-of-the-box Wi-Fi® connection successful")
+
+Now, you can click **OK** and be redirected to the Out-of-the-box homepage below.
+
+![Out-of-the-box Homepage](assets/OOTB_homepage.png "Out-of-the-box Homepage")
+
+***You can change your network by clicking on the Settings button and repeat the steps above.***
+
+#### Arduino Linux Wizard Homepage
+
+This web page is hosted on the Portenta X8 and allows a user to:
+
+- Get board details
+- [Configure Portenta X8 Wi-Fi®](#wi-fi-configuration)
+- [Interact with the board through the embedded Python® Alpine Shell](#portenta-x8-with-python-alpine-shell)
+- [Provision your device to Arduino Cloud](#portenta-x8-with-arduino-cloud)
+- Manage the Linux distribution with the dedicated [Portenta X8 Board Manager](#portenta-x8-board-manager)
+
```

Comment:
Include here that it is possible to update the board OS via the Arduino Linux Wizard

Reply:
Updated!